framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -499,6 +504,31 @@ public final synchronized void init(ProcessingEnvironment env) {
         }
     }
 
+    /** True if the git.properties file has been printed. */
+    private static boolean gitPropertiesPrinted = false;
+
+    /** Print information about the git repository from which the Checker Framework was compiled. */
+    void printGitProperties() {
```

Comment:
Should this method be private? Or protected? I don't see a need for package visibility.

Reply:
I made it private.